src/frontend/webpack.config.js

```diff
@@ -0,0 +1,100 @@
+const path = require("path");
+const HtmlWebPackPlugin = require("html-webpack-plugin");
+const webpack = require("webpack");
+
+require("dotenv").config({ path: "./.env" });
+
+module.exports = (env) => ({
+  mode: "development",
+  entry: "./src/index.js",
+  output: {
+    filename: "bundle.js",
+    path: path.resolve(__dirname, "build"),
+  },
+  resolve: {
+    modules: [path.join(__dirname, "src"), "node_modules"],
+    alias: {
+      react: path.join(__dirname, "node_modules", "react"),
+    },
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        include: path.resolve(__dirname, "src"),
+        use: [
+          {
+            loader: "babel-loader",
+            options: {
+              presets: [
+                [
+                  "@babel/preset-env",
+                  {
+                    targets: "defaults",
+                  },
+                ],
+                [
+                  "@babel/preset-react",
+                  {
+                    runtime: "automatic",
```

Comment:
By default, it is `runtime: "classic"`, but because we do not `import React from "react"` everywhere, we need to tell it to babel. See more here: https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#manual-babel-setup